GANA-UI.package/GanaBranchGraph.class/instance/initializeAlgorithm.st

```diff
@@ -0,0 +1,9 @@
+as yet unclassified
+initializeAlgorithm
```

Comment:
Does it really initialize the algorithm? I would say it initializes the data structures.